ui-admin/src/forms/designer/questions/ChoicesList.test.tsx

```diff
@@ -78,6 +78,28 @@ describe('ChoicesList', () => {
     })
   })
 
+  it('automatically generates values based on text', () => {
+    // Arrange
+    const onChange = jest.fn()
+    render(<ChoicesList question={question} readOnly={false} onChange={onChange} />)
+
+    const barChoice = screen.getAllByRole('listitem')[1]
+
+    // Act
+    const barLabelInput = getByLabelText(barChoice, 'Text')
+    fireEvent.change(barLabelInput, { target: { value: 'This is a test question' } })
```

Comment:
`fireEvent` is now deprecated.  New tests should use userEvent, which provides more realistic simulation of actual user input via a browser.  See https://testing-library.com/docs/user-event/intro


```suggestion
    userEvent.type(barLabelInput, 'This is a test question' )
```